deploy/helm/listener-operator/templates/roles.yaml

```diff
@@ -71,6 +71,7 @@ rules:
     resources:
       - listeners
       - listeners/status
+      - podlistenerses
```

Comment:
typo?

Reply:
Sadly no, the hobbitses are intentional in this case. `PodListeners` is already plural (it's a summary of many listener_s_), which must then be pluralized _again_ to get the resource name. The built-in `Endpoints` type has a similar problem.

Reply:
The plural name of the resource is used for the API resource path. `Endpoints` is already plural but its resource path is still `endpoints` and not `endpointses`. The same applies to `SecurityContextConstraints`. So `podlisteners` could also be used here as pseudo plural. With `PodListenerSet`, we would not have this issue. :)

Reply:
85673758f8a1fc323058b005a0d71bca455f8761